Detectors/CTP/simulation/src/Digits2Raw.cxx

```diff
@@ -39,12 +39,14 @@ void Digits2Raw::init()
   // Interaction Record
   int ilink = 0;
   uint64_t feeID = getFEEIDIR();
-  std::string outFileLink0 = mOutputPerLink ? o2::utils::Str::concat_string(outd, "ctp_link", std::to_string(ilink), ".raw") : o2::utils::Str::concat_string(outd, "ctp.raw");
+  //std::string outFileLink0 = mOutputPerLink ? o2::utils::Str::concat_string(outd, "ctp_link", std::to_string(ilink), ".raw") : o2::utils::Str::concat_string(outd, "ctp.raw");
+  std::string outFileLink0 = mOutputPerLink ? o2::utils::Str::concat_string(outd, "ctp_link", std::to_string(ilink), ".raw") : o2::utils::Str::concat_string(outd, mCTPRawDataFileName);
   mWriter.registerLink(feeID, mCruID, ilink, mEndPointID, outFileLink0);
   // Trigger Class record
   ilink = 1;
   feeID = getFEEIDTC();
-  std::string outFileLink1 = mOutputPerLink ? o2::utils::Str::concat_string(outd, "ctp_link", std::to_string(ilink), ".raw") : o2::utils::Str::concat_string(outd, "ctp.raw");
+  //std::string outFileLink1 = mOutputPerLink ? o2::utils::Str::concat_string(outd, "ctp_link", std::to_string(ilink), ".raw") : o2::utils::Str::concat_string(outd, "ctp.raw");
+  std::string outFileLink1 = mOutputPerLink ? o2::utils::Str::concat_string(outd, "ctp_link", std::to_string(ilink), ".raw") : o2::utils::Str::concat_string(outd, mCTPRawDataFileName);
```

Comment:
Could you please preserve the full name also in case of per-link output? 
e.g. define 
````
std::string mCTPRawDataFileName = "CTP_alio2-cr1-flp163_cru1111_0";
````
and compose the names as 
```
std::string outFileLink0 = mOutputPerLink ? fmt::format("{}{}_feeid{}.raw",outd, mCTPRawDataFileName, feeid) : fmt::format("{}{}.raw",outd, mCTPRawDataFileName, feeid);
```

Also, please remove obsolete commented code.

Reply:
Done.